specification/search/resource-manager/Microsoft.Search/preview/2019-10-01-preview/search.json

```diff
@@ -1106,6 +1417,190 @@
+          },
+          "description": "The list of Shared Private Link Resources."
+        }
+      },
+      "description": "Response containing a list of Shared Private Link Resources.",
+      "x-ms-external": true
+    },
+    "PrivateLinkResourcesResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
+          },
+          "description": "The list of available Private Link Resources."
+        }
+      },
+      "description": "Response containing a list of available Private Link Resources.",
+      "x-ms-external": true
+    },
+    "PrivateLinkResource": {
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The ID of the private link resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the private link resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type."
+        },
+        "properties": {
+          "$ref": "#/definitions/PrivateLinkResourceProperties",
+          "readOnly": true,
+          "description": "Describes the properties of an available private link resource for the Azure Cognitive Search service."
+        }
+      },
+      "description": "Describes an available private link resource for the Azure Cognitive Search service."
+    },
+    "PrivateLinkResourceProperties": {
+      "properties": {
+        "groupId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The group ID of the private link resource."
+        },
+        "requiredMembers": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The list of required members of the private link resource."
+        },
+        "requiredZoneNames": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The list of required DNS zone names of the private link resource."
+        },
+        "shareablePrivateLinkResourceTypes": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ShareablePrivateLinkResourceType"
+          },
+          "description": "The list of available shareable private link resource types."
+        }
+      },
+      "description": "Describes the properties of an available private link resource for the Azure Cognitive Search service."
+    },
+    "ShareablePrivateLinkResourceType": {
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the shareable private link resource type."
```

Comment:
let's expand on what "shareable" means in all the description fields succintly. Something like - 
shareable private link resource type <==> "resource type that has been onboarded to private link service, supported by Azure Cognitive Search."